x-pack/plugins/observability_solution/investigate_app/server/routes/get_global_investigate_app_server_route_repository.ts

```diff
@@ -138,6 +140,20 @@ const createInvestigationNoteRoute = createInvestigateAppServerRoute({
   },
 });
 
+const getAllInvestigationTagsRoute = createInvestigateAppServerRoute({
+  endpoint: 'GET /api/observability/investigations/_tags 2023-10-31',
```

Comment:
i will suggest to make this generic API to return all kinda term suggestiosn needed in the future instead of just tags.